config.json

```diff
@@ -1,1411 +1,1389 @@
-[{
-	"mavenRepositoryType" : "Google",
-	"slnFile" : "generated/GooglePlayServices.sln",
-	"debug": { "DumpModels" : true },
```

Comment:
Is this something we need?  If so I need to update the script to keep this.

Reply:
This:
```
"debug": { "DumpModels" : true },
```

Reply:
No we do not need this, but that does not hurt